kernel/configs/arm64_kata_kvm_4.19.x

```diff
@@ -500,6 +502,7 @@ CONFIG_ACPI_TABLE_UPGRADE=y
 # CONFIG_ACPI_DEBUG is not set
 # CONFIG_ACPI_PCI_SLOT is not set
 CONFIG_ACPI_CONTAINER=y
+CONFIG_ACPI_HOTPLUG_MEMORY=y
```

Comment:
I presume the HOTPLUG additions are deliberate. May want to add a note to the commit message, as they look like not just part of an update, but are an addition.

Reply:
yep, you just got me😱. I may need to split that to another commit...🤠

Reply:
There are several entries added in this patch. Do you have any explanation here @Pennyzct?